projects/scion/workbench/src/lib/dialog/ɵworkbench-dialog.ts

```diff
@@ -314,10 +334,23 @@ export class ɵWorkbenchDialog<R = unknown> implements WorkbenchDialog<R> {
   private blockWhenNotOnTop(): void {
     this._workbenchDialogRegistry.top$({viewId: this.context.view?.id})
       .pipe(
-        map(top => top !== this),
+        map(top => top === this ? null : top),
+        takeUntilDestroyed(this._destroyRef),
+      )
+      .subscribe(this.blockedBy$);
+  }
+
+  /**
+   * Blinks briefly when requested.
```

Comment:
Do we need the comment? It doesn't give more information than the name, does it?